truss/truss_config.py

```diff
@@ -297,19 +297,67 @@ def to_list(self) -> List[Dict[str, str]]:
         return [item.to_dict() for item in self.items]
 
 
+class DockerAuthType(Enum):
+    GCS_SERVICE_ACCOUNT_JSON_BASE64 = "GCS_SERVICE_ACCOUNT_JSON_BASE64"
+    ACCESS_TOKEN = "ACCESS_TOKEN"
```

Comment:
The access token is also technically, username:access_token | base64. If you're going to write the encoding in the name, maybe it's worth making that explicit here?